server/endpoints/AdminChart.ts

```diff
@@ -187,7 +187,7 @@ const topSubjectsCB = async (_ctx: Context) => {
  * Returns the top 15 subjects (in terms of number of reviews)
  */
 export const topSubjects: Endpoint<unknown> = {
-  guard: [body("token").notEmpty()],
+  guard: [],
```

Comment:
Why did you remove the guard?

Reply:
When I first migrated the backend topSubjects method I thought it did take in an auth token (thus i put it in the guard) but it turns out that it does not take in anything (so i removed it in this PR). Should we alter the topSubjects method to require a token? (I think that makes sense too).

Reply:
If we don't already, we should probably gate all the admin functions behind tokens.